New pages: HTMLMeterElement properties (#35638)
mdn/content: files/en-us/web/api/htmlmeterelement/high/index.md

---
title: "HTMLMeterElement: high property"
short-title: high
slug: Web/API/HTMLMeterElement/high
page-type: web-api-instance-property
browser-compat: api.HTMLMeterElement.high
---

{{ APIRef("HTML DOM") }}

> The **`high`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the high boundary of the {{htmlelement("meter")}} element as a `double`. It reflects the element's [`high`](/en-US/docs/Web/HTML/Element/meter#high) attribute, or the value of `max` if not defined. The value of `high` is clamped by the `low` and `max` values.

Reviewer: ```suggestion
The **`high`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the high boundary of the {{htmlelement("meter")}} element as a floating-point number. It reflects the element's [`high`](/en-US/docs/Web/HTML/Element/meter#high) attribute, or the value of `max` if not defined. The value of `high` is clamped by the `low` and `max` values.
```

This property can also be set directly, for example to set a default value based on some condition.

## Value

> A number. The value of `high`, but not less than {{DOMxRef("HTMLMeterElement.low")}} nor greater than {{DOMxRef("HTMLMeterElement.max")}}.

Reviewer: ```suggestion
A number that is not less than {{DOMxRef("HTMLMeterElement.low")}} nor greater than {{DOMxRef("HTMLMeterElement.max")}}.
```
